### PR TITLE
Add YouTube links to community-resources.json

### LINKED ISSUE #9037: [OTHER] Add Kim's Community Resource
### What would you like to share?

- Update the json file https://github.com/EddieHubCommunity/BioDrop/blob/main/config/community-resources.json
- YouTube link 
   - https://twitter.com/kamcmahon/status/1701276223090073865
   - https://www.youtube.com/watch?v=LAsz48CixZY
- So it appears on the community docs page https://www.biodrop.io/docs/community-resources

### Do you want to work on this issue?

Going to be assigned to https://twitter.com/Nene_dev26

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Nenepo**
fixes #9037

### CODE REVIEW

File: config/community-resources.json [hunk 1]
```diff
@@ -98,5 +98,15 @@
       "name": "Maximising Your Job Prospects in Tech with BioDrop",
       "url": "https://loftwah.medium.com/maximising-your-job-prospects-in-tech-with-BioDrop-2fea408301cc"
     }
+  },
+  {
+    "name": "Nafisa Faruna",
```
Comment (eddiejaoude) [inline config/community-resources.json:103]: The name here should be the creator of the resource

```suggestion
    "name": "Kim McMahon",
```

**Comment by Nenepo** [comment]
Oh alright
I will fix it

File: config/community-resources.json [hunk 2]
```diff
@@ -98,5 +98,15 @@
...
+    "username": "Nenepo",
```
Comment (eddiejaoude) [inline config/community-resources.json:104]: This should be the username of the resource author 

```suggestion
    "username": "KimMcMahon",
```

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for the contribution 👍 

I have left 2 small inline comments

**Comment by Nenepo** [comment]
Thanks for the feedback!

**Comment by eddiejaoude** [comment]
No worries, you can push more commits to your fork branch and GitHub will add it to this open pull request

**Comment by eddiejaoude** [comment]
@Nenepo do you need help with making these changes?

**Comment by Nenepo** [comment]
Hyyyyy
Not at the moment, I would have to go back and check it out
I did the push already and sent in an email
I had no idea it did not work
Sorry about that

**Comment by Nenepo** [comment]
> @Nenepo do you need help with making these changes?

Hyyyyy I realized I didn’t open a pull request
I just opened it

**Comment by eddiejaoude** [comment]
> No worries, you can push more commits to your fork branch and GitHub will add it to this open pull request

I will close this PR as you have created a new one https://github.com/EddieHubCommunity/BioDrop/pull/9153

**Comment by Nenepo** [comment]
Thank you,I had to open another branch because I kept pushing and it was
not reflecting on GitHub but I could see the commit on my local server
I got tired of trying to figure it out😫

**Pull request closed without merging**